modules/log6x-collection-setup.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-collection-setup_{context}"]
+= Setting up log collection
+
+This release of Cluster Logging requires administrators to explicitly grant log collection permissions to the service account associated with *ClusterLogForwarder*. This was not required in previous releases for the legacy logging scenario consisting of a *ClusterLogging* and, optionally, a *ClusterLogForwarder.logging.openshift.io* resource.
+
+The {clo} provides `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles, which enable the collector to collect audit logs, application logs, and infrastructure logs respectively.
```

Comment:
There also is `logging-collector-logs-writer` which encompasses all three roles. Do we want to mention this one as well? (cc @cahartma @jcantrill )

Reply:
I agree.  This seems to me like several sections that could be covered more simply.   Most of this is kube concepts that we shouldn't need to repeat, possibly just link to.   
Copying my comment here from the "roles" section: 

We may consider adding something similar to Monitoring, where we list a table of roles, then additional links to OCP docs on how to grant permissions. https://docs.openshift.com/container-platform/4.16/observability/monitoring/configuring-the-monitoring-stack.html#granting-users-permissions-for-core-platform-monitoring_configuring-the-monitoring-stack